br/pkg/task/show/cmd_test.go

```diff
@@ -178,6 +178,7 @@ func TestShowViaSQL(t *testing.T) {
 	err := os.WriteFile(metaPath, FullMeta, 0o444)
 	req.NoError(err)
 
+	tk.MustExec("set @@time_zone='+08:00'")
```

Comment:
please also add other time_zone to verify the output will change.

Reply:
Done